include/rocksdb/utilities/options_type.h

```diff
@@ -75,7 +75,12 @@ enum class OptionTypeFlags : uint32_t {
   kCompareExact = ConfigOptions::kSanityLevelExactMatch,
 
   kMutable = 0x0100,        // Option is mutable
+  kPointer = 0x0200,        // The option is stored as a pointer
```

Comment:
Can we make it clear: kPointer->kRawPointer (if I understand correctly. Or it is any pointer type including shared and unique?); kShared->kSharedPointer; kUnique->kUniquePointer?